example/FilesApiExample.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2020 Smartling, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this work except in compliance with the License.
+ * You may obtain a copy of the License in the LICENSE file, or at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.smartling.api.example;
```

Comment:
Is it ok that this won't compile?

Reply:
removed package
but it still won't compile without dependencies
it is supposed to be added into a configured project